ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/baseActivity.component.ts

```diff
@@ -6,10 +6,16 @@ import {
     OnDestroy,
     Output,
     SimpleChange,
-    Component
+    Component,
```

Comment:
_**#minor**_

we don't support/use trailing commas.

Reply:
![image](https://user-images.githubusercontent.com/98110251/156513270-847b0bed-fd27-4b45-b365-e627d0254db1.png)
I was going to disable this but it seems like this is a es5 feature, commas are removed through minification